code/modules/mob/typing_indicator.dm

```diff
@@ -1,31 +1,33 @@
-/mob/proc/toggle_typing_indicator(emoting)
-	var/image/typing_indicator
+/mob/proc/add_typing_indicator(emoting)
+	if(stat != CONSCIOUS)
+		return
+
+	var/image/I
```

Comment:
Since `typing_indicator` is type cast, this intermediary var is not really necessary.